contracts/test/PoolDelegate.t.sol

```diff
@@ -268,7 +296,10 @@ contract PoolTest is TestUtil {
 
     }
 
-    function test_deactivate_fail() public {
+    function test_deactivate_fail(
+        uint256 depositAmt,
+        uint256 fundAmt
+    ) public {
```

Comment:
One line

Reply:
I think we should actually leave this as a unit test, don't really need to fuzz this.

Reply:
It is a cherry on top, So no harm